yarn.lock

```diff
@@ -3518,6 +3518,7 @@ __metadata:
   resolution: "@metamask/storybook-react@workspace:apps/storybook-react"
   dependencies:
     "@chromatic-com/storybook": "npm:^1.9.0"
+    "@metamask/design-system-react": "workspace:^"
```

Comment:
Because we are using design-system-react components in the storybook-react app we need to update our dependencies accordingly